### PR TITLE
Sort more

### PR DESCRIPTION
**Pull request by Donovan9617**
Sort client by name
Sort issue by status of issue count 0 for incomplete and 1 for complete 
Fixed a bug with project edit to reflect changes in client

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -156,15 +156,17 @@ public interface Model {
...
     void updateFilteredClientList(Predicate<Client> predicate);
 
-    void sortProjectsByDeadline(int sortByDeadlineKey);
+    void sortProjectsByDeadline(int sortProjectsByDeadlineKey);
 
-    void sortProjectsByIssueCount(int sortByIssueCountKey);
+    void sortProjectsByIssueCount(int sortProjectsByIssueCountKey);
 
-    void sortProjectsByName(int sortByNameKey);
+    void sortProjectsByName(int sortProjectsByNameKey);
```
Comment (Aishwarya-Hariharan-Iyer) [inline src/main/java/seedu/address/model/Model.java:137]: Nice job renaming the variables for clarity!

File: src/main/java/seedu/address/logic/commands/client/SortClientCommand.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class SortClientCommand extends ClientCommand {
+
+    public static final String COMMAND_FLAG = "-s";
+
+    public static final String MESSAGE_SUCCESS = "Sorted clients";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sort clients in address book. "
+            + "Sort by name: "
+            + PREFIX_CLIENT_NAME + "0 (alphabetical) or "
+            + PREFIX_CLIENT_NAME + "1 (reverse alphabetical). "
```
Comment (Aishwarya-Hariharan-Iyer) [inline src/main/java/seedu/address/logic/commands/client/SortClientCommand.java:25]: Just a general observation: 
Might be better to standardise as PREFIX_NAME for all, or have PREFIX_CLASS_NAME for all (CLASS here is our model classes).

Comment (Donovan9617) [reply src/main/java/seedu/address/logic/commands/client/SortClientCommand.java]: Standardised codebase to have `PREFIX_NAME`

**Review by crvstalphua — APPROVED** [review]
LGTM

**Pull request merged**